src/plugins/mce/plugin.c

```diff
@@ -92,10 +117,10 @@ mce_signal_filter (NCore *core, DBusConnection *connection, DBusMessage *msg, vo
             N_DEBUG (LOG_CAT "%s >> mce finished playing %s", __FUNCTION__, pattern);
             for (event = active_events; event != NULL; event = g_list_next(event)) {
                 MceData *data = (MceData *) event->data;
-                if (g_strcmp0(pattern, data->pattern) == 0) {
+                if (g_strcmp0 (pattern, data->pattern) == 0 && data->playing) {
+                    active_events = g_list_remove_all(active_events, data);
```

Comment:
Nitpick this has no space before ( but then again this whole file has a bit inconsistent use of space before ( in function calls.